src/integration/pages/homePage/kyrgyz/__snapshots__/amp.test.js.snap

```diff
@@ -207,10 +207,7 @@ Object {
 
 exports[`AMP Home Page Main heading should match text 1`] = `"BBC News, Кыргыз КызMATы - Башталгыч бет"`;
 
-exports[`AMP Home Page Page content First item in the first curation is the correct headline 1`] = `
-"13:31 - “Баланы сабады деп аялымды камап салышарын өзүм да билген эмесмин”. Кадамжайда токмоктолгон наристенин атасы
-"
-`;
+exports[`AMP Home Page Page content First item in the first curation is the correct headline 1`] = `"Article tagged with Trump"`;
 
 exports[`AMP Home Page SEO Apple Touch Icon should match attributes 1`] = `
```

Comment:
Is amp still used for home pages?

Reply:
We were able to get amp working without errors for the new homepage, so have decided to keep it for now. These amp integration tests are particularly handing for helping us with the analytics - makes sure that the correct values are being set 🙌